web/sites/default/config/views.view.taxonomy_child_terms.yml

```diff
@@ -143,7 +143,7 @@ display:
             title_enable: false
             title: All
           title_enable: true
-          title: 'Topics in {{ arguments.parent_target_id }}'
+          title: '{{ ''Topics in''|t }} {{ arguments.parent_target_id }}'
```

Comment:
Should this be a single quote?

Reply:
No, the string is wrapped in single quotes so you have to escape the single quote with `''` otherwise it interprets the quote as the end of the string.